types/react-linkify/index.d.ts

```diff
@@ -7,7 +7,7 @@
 
 import * as React from "react";
 
-declare namespace ReactLinkify {
+export namespace ReactLinkify {
```

Comment:
thanks for bringing this up, this currently works for `import Linkify from "react-linkify";` and 
```
<Linkify componentDecorator={decoratorOverride}>
              // some text
</Linkify>
```
wondering why you need such namespace being exported?